test/Parameters/test_parameters.py

```diff
@@ -0,0 +1,34 @@
+import BioSimSpace as BSS
+
+import os
+import pytest
+
+# Make sure required AMBER executables are present.
+if BSS._amber_home is not None:
+    tleap = "%s/bin/tleap" % BSS._amber_home
+    if os.path.isfile(tleap):
+        has_tleap = True
+    else:
+        has_tleap = False
+else:
+    has_tleap = False
+
+@pytest.mark.skipif(has_tleap is False, reason="Requires tLEaP to be installed.")
+def test_disulphide():
+    """Test parameterisation in the presence of disulphide bridges."""
+
+    # Load the example molecule.
+    molecule = BSS.IO.readMolecules("test/input/molecules/4LYT_Fixed.pdb")[0]
+
+    # Try to parameterise with ff14SB. If working, this should auto-detect
+    # disulphide bonds and add the appropriate bond records to the tLEaP input
+    # script.
+    molecule = BSS.Parameters.ff14SB(molecule).getMolecule()
```

Comment:
If we need to keep the separate force fields duplicate, I'd `pytest.mark.parametrize` this with every supported force field (I think I would probably do it even if it was abstracted, just to be safe).